src/box-client.ts

```diff
@@ -634,67 +628,6 @@ class BoxClient {
 		return this._makeRequest(newParams, callback);
 	}
 
-	/**
-	 * Puts the client into batch mode, which will queue calls instead of
-	 * immediately making the API request.
-	 *
-	 * DEPRECATED: Batch API is not supported and should not be used; make calls in parallel instead.
-	 *
-	 * @returns {BoxClient} Current client object
-	 */
-	batch = util.deprecate(function (this: BoxClient) {
-		/* eslint-disable no-invalid-this */
-		this._batch = [];
-		return this;
-		/* eslint-enable no-invalid-this */
-	}, 'Batch API is not supported and should not be used; make calls in parallel instead.');
-
-	/**
-	 * Executes a batch of requests.
-	 *
-	 * DEPRECATED: Batch API is not supported and should not be used; make calls in parallel instead.
-	 *
-	 * @returns {Promise<Object>} Promise resolving to the collection of batch responses
-	 */
-	batchExec = util.deprecate(function (this: BoxClient, callback: Function) {
-		/* eslint-disable no-invalid-this */
-		if (!this._batch) {
-			return Promise.reject(
-				new Error('Must start a batch before executing')
-			).asCallback(callback);
-		}
-
-		var params = {
-			body: {
-				requests: this._batch.map((batchReq: any /* FIXME */) =>
-					formatRequestForBatch(batchReq.params)
```

Comment:
Can you also remove all that is currently not used since you deleted those `batch` method, like:
`formatRequestForBatch` or `_batch`, and maybe more?